pkg/env/env.go

```diff
@@ -4,39 +4,26 @@
 package env
 
 import (
-	"context"
-	"log"
 	"os"
 	"path"
+	"runtime"
 	"strings"
-
-	execute "github.com/alexellis/go-execute/v2"
 )
 
 // GetClientArch returns a pair of arch and os
 func GetClientArch() (arch string, os string) {
-	task := execute.ExecTask{
-		Command:     "uname",
```

Comment:
I'm sorry but this is not what I was asking for.

There should be a separate version of GetClientArch() which is combined only for Windows which looks at runtime.GOOS and returns the correct value. It could even be hard-coded given that it will only build in for Windows.

Then leave the existing code as it is for "unix"

Here's an example of how it works in Go: https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool

Reply:
Ahh, I must have misunderstood, will fix asap! :)